### PR TITLE
add ipv6 shoot manifest to docs

### LINKED ISSUE #1134: Add IPv6 Shoot example to the usage docs
**How to categorize this issue?**

/area networking
/kind enhancement
/platform aws

**What would you like to be added**:
https://github.com/gardener/gardener-extension-provider-aws/blob/master/docs/usage/usage.md contains the following Shoot examples - single zone Shoot, multiple zone Shoot.
I would be great if Shoot using ipv6 example is added.

**Why is this needed**:
Document the new features so that end users can create ipv6 Shoot without issues.

### PR DESCRIPTION
**Pull request by DockToFuture**
**How to categorize this PR?**

/area networking
/kind enhancement
/platform aws

**What this PR does / why we need it**:
Add IPv6 shoot manifest to docs.

**Which issue(s) this PR fixes**:
Fixes #1134

**Special notes for your reviewer**:

**Release note**:

```other operator
none
```

### CODE REVIEW

**Review by ScheererJ — DISMISSED** [review]
Thanks for adding an example documentation.

/lgtm

**Review by hebelsan — COMMENTED** [review]
In general looks good to me.
Can we though remove the workerConfig part since it's not related to this example?

**Comment by DockToFuture** [comment]
> In general looks good to me. Can we though remove the workerConfig part since it's not related to this example?

The whole workers section?

File: docs/usage/usage.md [hunk 1]
```diff
@@ -595,6 +595,53 @@ spec:
...
+      networks:
+        vpc:
+          cidr: 10.250.0.0/16
+        zones:
+        - name: eu-central-1a
+          public: 10.250.96.0/22
+          internal: 10.250.112.0/22
+    controlPlaneConfig:
+      apiVersion: aws.provider.extensions.gardener.cloud/v1alpha1
+      kind: ControlPlaneConfig
```
Comment (ScheererJ) [inline docs/usage/usage.md:627]:
```suggestion
      kind: ControlPlaneConfig
    workers:
    - ...
```

File: docs/usage/usage.md [hunk 2]
```diff
@@ -595,6 +595,53 @@ spec:
...
+  networking:
+    ipFamilies:
+    - IPv6
+    type: calico
+  kubernetes:
+    version: 1.28.2
+  maintenance:
+    autoUpdate:
+      kubernetesVersion: true
+      machineImageVersion: true
```
Comment (ScheererJ) [inline docs/usage/usage.md:637]:
```suggestion
  ...
```

**Review by hebelsan — APPROVED** [review]
/lgtm

**Pull request merged**